### PR TITLE
Additional Map Tests

### LINKED ISSUE #8145: Missing Map tests
[Map](https://leafletjs.com/SlavaUkraini/reference.html#map-setview):
- [x] setZoom
- [x] setZoomAround
- [x] fitWorld
- [x] #8365
- [ ] #8366
- [ ] #8367
- [x] setMinZoom
- [x] setMaxZoom
- [ ] #8368
- [ ] #8369
- [ ] #8370
- [ ] #8371
- [ ] #8372
- [x] createPane
- [x] getPane
- [x] getPanes
- [x] getContainer
- [x] getSize
- [x] getPixelBounds
- [x] getPixelOrigin
- [x] getPixelWorldBounds
- [ ] #8373
- [ ] #8374
- [ ] #8375
- [x] #8376
- [x] distance
- [x] containerPointToLayerPoint
- [x] layerPointToContainerPoint
- [x] #8377
- [x] #8378
- [ ] #8379
- [ ] #8380
- [ ] #8381

### PR DESCRIPTION
**Pull request by stephenspol**
Unit tests missed for `Map`. Closes part of #8145.

- [x] `setZoom`
- [x] `setZoomAround`
- [x] `fitWorld`
- [ ] `panTo`
- [ ] `panBy`
- [ ] `flyToBounds`
- [x] `setMinZoom`
- [x] `setMaxZoom`
- [ ] `panInsideBounds`
- [ ] `stop`
- [ ] `locate`
- [ ] `stopLocate`
- [ ] `addHandler`
- [x] `createPane`
- [x] `getPane`
- [x] `getPanes`
- [x] `getContainer`
- [x] `getSize`
- [x] `getPixelBounds`
- [x] `getPixelOrigin`
- [x] `getPixelWorldBounds`
- [ ] `project`
- [ ] `unproject`
- [ ] `layerPointToLatLng`
- [ ] `latLngToLayerPoint`
- [x] `distance`
- [x] `containerPointToLayerPoint`
- [x] `layerPointToContainerPoint`
- [ ] `containerPointToLatLng`
- [ ] `latLngToContainerPoint`
- [ ] `mouseEventToContainerPoint`
- [ ] `mouseEventToLayerPoint`
- [ ] `mouseEventToLatLng`

Some options such as `animate` and `maxZoom` are not covered as I don't know the functionality as examples are missing in the source. Also, please let me know if I miss any functionality or known bugs.

### CODE REVIEW

File: spec/suites/map/MapSpec.js
Comment (stephenspol) [inline spec/suites/map/MapSpec.js:561]: Not sure if the intended function is to move to the appropriate zoom or actual change the zoom value.

```
if (this.getZoom() < this.options.minZoom) {
	return this.setZoom(zoom);
}
```

**Comment by jonkoops** [comment]
@stephenspol Thanks for the PR, it's much appreciated! Could you do me a solid and try to split this work into separate PRs for each method to reduce the cognitive load of the review?

**Review by Falke-Design — CHANGES_REQUESTED** [review]
@jonkoops I reviewed the tests and for me it would be ok to keep the already implmented tests  and merge this PR (if no new tests will be added). Is that ok for you too?

@stephenspol can you please fix the requested changes and all the `.skip` tests. After that I would like to merge this PR as soon as possible.

**Comment by stephenspol** [comment]
Sorry for the sudden stop of progress on this PR but I have fixed the recommended changes. Please let me know if any test names or test functionalities need updated.

**Comment by jonkoops** [comment]
@Falke-Design fair enough, we can certainly land this if you'd like. Let me take a look and see if this all makes sense.

**Review by Falke-Design — APPROVED** [review]

**Comment by Falke-Design** [comment]
@stephenspol thank you for your contribution! If you want to implement more test, please bundle them into smaller PRs.

**Pull request merged**
